packages/babel-helper-member-expression-to-functions/src/index.js

```diff
@@ -306,11 +306,12 @@ const handle = {
         // assignment.
         this.memoise(member, 2);
 
-        value = t.binaryExpression(
-          operator.slice(0, -1),
-          this.get(member),
-          value,
-        );
+        const operatorTrunc = operator.slice(0, -1);
+        if (t.LOGICAL_OPERATORS.includes(operatorTrunc)) {
```

Comment:
Ooo, can we fix `https://github.com/babel/babel/blob/master/packages/babel-plugin-proposal-logical-assignment-operators/src/index.js#L16`, too?

Reply:
Will do.